test/cppsuite/tests/test_live_restore.cpp

```diff
@@ -363,24 +364,30 @@ run_restore(const std::string &home, const std::string &source, const int64_t th
     auto crud_session = connection_manager::instance().create_session();
     if (recovery)
         configure_database(crud_session);
-    if (!background_thread_mode)
-        do_random_crud(crud_session, collection_count, op_count, false, conn_config, home);
+    do_random_crud(crud_session, collection_count, op_count, false, conn_config, home);
     if (die)
         raise(SIGKILL);
 
     // Loop until the state stat is complete!
-    if (thread_count > 0 || background_thread_mode) {
-        logger::log_msg(LOG_INFO, "Waiting for background data transfer to complete...");
-        while (true) {
-            auto stat_cursor = crud_session.open_scoped_cursor("statistics:");
-            int64_t state;
-            get_stat(stat_cursor.get(), WT_STAT_CONN_LIVE_RESTORE_STATE, &state);
-            if (state == WT_LIVE_RESTORE_COMPLETE)
-                break;
-            __wt_sleep(1, 0);
-        }
-        logger::log_msg(LOG_INFO, "Done!");
+    logger::log_msg(LOG_INFO, "Waiting for background data transfer to complete...");
+    while (true) {
+        auto stat_cursor = crud_session.open_scoped_cursor("statistics:");
+        int64_t state;
+        get_stat(stat_cursor.get(), WT_STAT_CONN_LIVE_RESTORE_STATE, &state);
+        if (state == WT_LIVE_RESTORE_COMPLETE)
+            break;
+        __wt_sleep(1, 0);
     }
+    logger::log_msg(LOG_INFO, "Done!");
+
+    // Once background migration has finished there's no reason for us to access the backing source
```

Comment:
note: s_all is failing because this `//` comment is three lines long. They style guide allows `//` comments up to two lines, but 3 line comments need to be wrapped in `/* */`